### PR TITLE
Updated DB size query as per latest etcd versions

### PR DESCRIPTION
**Pull request by RagalahariP**
# Summary

Updated DB size query as per latest etcd versions
Reference - https://etcd.io/docs/v3.4/upgrades/upgrade_3_5/

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Are all documentation links publically accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [x] Did you check your descriptive content for spelling and grammar errors?

### CODE REVIEW

File: dashboards/etcd/etcd.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
               "nrqlQueries": [
                 {
                   "accountId": 0,
-                  "query": "SELECT (average(etcd_debugging_mvcc_db_total_size_in_bytes) * cardinality(etcd_debugging_mvcc_db_total_size_in_bytes)) / 1000 / 1000  as 'DB Total Size (MB)' FROM Metric where instrumentation.name = 'remote-write' facet instance SINCE 60 MINUTES AGO UNTIL NOW"
+                  "query": "SELECT (average(etcd_mvcc_db_total_size_in_bytes) * cardinality(etcd_mvcc_db_total_size_in_bytes)) / 1000 / 1000  as 'DB Total Size (MB)' FROM Metric where instrumentation.name = 'remote-write' facet instance SINCE 60 MINUTES AGO UNTIL NOW"
```
Comment (thezackm) [inline dashboards/etcd/etcd.json:125]: i would update this to make it backwards compatible with older versions:

```
SELECT 
(average(etcd_mvcc_db_total_size_in_bytes) * cardinality(etcd_mvcc_db_total_size_in_bytes)) 
OR (average(etcd_debugging_mvcc_db_total_size_in_bytes) * cardinality(etcd_debugging_mvcc_db_total_size_in_bytes)) / 1000...
```

please check the syntax on this, but the logic is there

File: dashboards/etcd/etcd.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
+              "text": "![New Relic logo](https://i.imgur.com/zQS8meU.png) \n- - -\n### Overview\n`etcd` is a strongly consistent, distributed key-value store that provides a reliable way to store data that needs to be accessed by a distributed system or cluster of machines. It gracefully handles leader elections during network partitions and can tolerate machine failure, even in the leader node.\n\n### About this Dashboard\n\nThis dashboard is based on standard `etcd` Prometheus metrics ingested using Prometheus [remote write](https://newrelic.com/instant-observability/prometheus-remote-write/7dbd7355-c380-4211-bd32-fed9a65351af).  Most widgets utilize an `instance` facet to identify `etcd` instances in the cluster.  \n\n**Note**:  if you're running a managed Kubernetes cluster (e.g. EKS, AKS, GKE), then the full set of `etcd` Prometheus metrics will not be available in these platforms and DB Total Size (MB) widget is supported by etcd 3.5 and later versions. \n\n"
```
Comment (thezackm) [inline dashboards/etcd/etcd.json:22]: based on notes below for the query, you may want to update this to account for older versions too

**Review by thezackm — CHANGES_REQUESTED** [review]
@RagalahariP - please see notes

File: dashboards/etcd/etcd.json (hunk 1 above)
Comment (RagalahariP) [reply dashboards/etcd/etcd.json]: Thank you @thezackm , updated query as suggested.

image

**Review by thezackm — APPROVED** [review]
approved for the `prometheus-growth` project

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**